.github/workflows/build-no-cache.yml

```diff
@@ -44,12 +44,24 @@ jobs:
           username: ${{ github.repository_owner }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Build docker image
+      - name: Build Base
         uses: docker/build-push-action@v3.1.1
         with:
+          context: .
+          tags: ${{ env.DOCKER_REPOSITORY }}:base-master
+          push: true
+          target: base
+          build-args: |
+            SHA=${{ steps.sha.outputs.short }}
+            BUILD_TYPE=release
+
+      - name: Build and push to Docker Hub
```

Comment:
We use `push: false`. Should the title be something like "Build docker image locally" ?

Reply:
changed title